gems/smithy/spec/fixtures/shapes/model.smithy

```diff
@@ -0,0 +1,115 @@
+$version: "2"
+
+namespace smithy.ruby.tests
+
+@paginated(inputToken: "nextToken", outputToken: "nextToken", pageSize: "pageSize")
+service ShapeService {
+    version: "2018-10-31"
+    resources: [
+        SomeResource
+    ]
+    operations: [
+        SomeOperation
+    ]
+}
+
+operation SomeOperation {
+    input: SomeOperationInput
+    output: SomeOperationOutput
```

Comment:
I think you'll want something more kitchen sink for both input and output. SomeOperationInputOutput probably where it has some structure that contains all of the shapes. It should also reference itself, which would make it nested/recursive. We will eventually need it on both input and output for building and parsing.